jdaviz/configs/specviz/helper.py

```diff
@@ -150,7 +150,11 @@ def y_limits(self, y_min=None, y_max=None):
         ref_index = self.app.get_viewer(
             self._default_spectrum_viewer_reference_name
         ).state.reference_data.label
-        flux_axis = self.get_spectra(ref_index).flux
+        if self.app.config == "cubeviz":
+            rs_opt = False
+        else:
+            rs_opt = True
```

Comment:
I would expect this to also happen in specviz and that we should always pass `False` here, no?

Reply:
I dunno... What does True or False really mean?

Reply:
The only reason is I don't know what is supposed to happen in other configs. So I am hoping this change will suppress warning but does not change any existing behavior. If you tell me setting to False always is safe, I will make that change. Please confirm. Thanks!